src/imfilter_cuda.jl

```diff
@@ -0,0 +1,41 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENCE in the project root.
+# ------------------------------------------------------------------
+
+function imfilter_resource(resource::CUDALibs{R},
+                         img::AbstractArray{T,N},
+                         kern::AbstractArray{K,N}) where {R, T<:Real,K<:Real,N}
+
+  sizeall = Tuple(map(t->t[1]+t[2]-1,zip(size(img),size(kern))))
```

Comment:
You are still dealing with a code that 2-dimensional. Can you please explain why you mark my comment as resolved but didn't change the code?

Reply:
I found the comment about dimensionaly and replied there. The code is N-dimensional.

Reply:
Much better. More declarative code.